packages/react-data-grid/src/Cell.tsx

```diff
@@ -159,31 +160,96 @@ export default class Cell<R> extends React.Component<CellProps<R>> implements Ce
     return allEvents;
   }
 
+  handleDeleteSubRow = (): void => {
+    const { idx, rowIdx, expandableOptions, cellMetaData, rowData } = this.props;
+
+    const { onDeleteSubRow } = cellMetaData;
+
+    if (onDeleteSubRow) {
+      onDeleteSubRow({
+        idx,
+        rowIdx,
+        rowData,
+        expandArgs: expandableOptions
+      });
+    }
+  }
+
   render() {
-    const { idx, rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling } = this.props;
+    const { rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling, isSummaryRow } = this.props;
     if (column.hidden) {
       return null;
     }
 
     const style = this.getStyle();
-    const className = this.getCellClass();
-    const cellContent = children || (
-      <CellContent<R>
-        idx={idx}
-        rowIdx={rowIdx}
-        column={column}
-        rowData={rowData}
-        value={value}
-        tooltip={tooltip}
-        expandableOptions={expandableOptions}
-        height={height}
-        onDeleteSubRow={cellMetaData.onDeleteSubRow}
-        cellControls={cellControls}
-        isScrolling={isScrolling}
-        isRowSelected={this.props.isRowSelected}
-        onRowSelectionChange={this.props.onRowSelectionChange}
+    const cellClassName = this.getCellClass();
+    const cellValueClassName = classNames('react-grid-Cell__value', { 'cell-tooltip': !!tooltip });
+
+    if (isSummaryRow) {
+      return (
+        <div
+          ref={this.cell}
+          className={cellClassName}
+          style={style}
+        >
+          <div className={cellValueClassName}>
+            <div className="react-grid-Cell__container">
+              <span>
+                <CellValue<R>
+                  rowIdx={rowIdx}
+                  rowData={rowData}
+                  column={column}
+                  value={value}
+                  isScrolling={isScrolling}
+                  isRowSelected={this.props.isRowSelected}
+                  onRowSelectionChange={this.props.onRowSelectionChange}
+                  isSummaryRow={isSummaryRow}
+                />
+              </span>
+            </div>
+            {tooltip && <span className="cell-tooltip-text">{tooltip}</span>}
```

Comment:
I think we should deprecate `tooltip` prop. `Row/Cell` renderers must be used to set it and it makes using grid difficult IMO. We should just set the tooltip in the formatter. We do not have to remove `tooltip` prop in this PR but we can remove it from `summaryRow`. MediaPlan extensively use this prop so we have to keep in on the regular row for now

Reply:
I believe we need to update some styles as well. (we should do this in another PR). Currently if we set the tooltip in the formatter, the tooltip will only display when the formatter area is hovered but not the whole cell.

Reply:
I was referring to that we should remove `{tooltip && <span className="cell-tooltip-text">{tooltip}</span>}` from this block. We do not need to handle tooltip on the totals row